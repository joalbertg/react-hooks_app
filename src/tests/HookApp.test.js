import React from 'react';
import { shallow } from 'enzyme';

import { HookApp } from '../HookApp';

describe('Tests HookApp component', () => {
  test('Should display correctly', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});

