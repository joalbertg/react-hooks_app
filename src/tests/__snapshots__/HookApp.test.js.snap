// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests HookApp component Should display correctly 1`] = `
<Fragment>
  <h1>
    Hello World
  </h1>
</Fragment>
`;
